### DISCUSSION TITLE
Documentation for enabling aws-iot-device-client features

### DISCUSSION THREAD

**Discussion by bawilless** [aws4embeddedlinux/meta-aws#1563] [Q&A] [answered]
I'm new to yocto and leveraging meta layers.  Can the team add some simple documentation to the meta-aws/recipes-iot/aws-iot-device-client folder with instructions for enabling a feature such as the aws-iot-device-client PubSub feature?

I can see the PACKAGECONFIG[] entries in the *.bb file, but I don't know what to do with them to enable the samples and pubsub features.

Thanks!

**Comment by thomas-roos** [comment, marked as answer]
Hi,
PACKAGECONFIG is a yocto common thing documented here: https://docs.yoctoproject.org/ref-manual/variables.html?highlight=packageconfig#term-PACKAGECONFIG
-> to enable samples and pubsub feature you can put 
PACKAGECONFIG:pn-aws-iot-device-client = "samples pubsub"
into your local.conf
Hope this helps.

**Comment by bawilless** [comment]
Thanks for your support Thomas.  I'm still trying to get this working, but you've pointed me in the right direction.  This discussion can be closed.
